catalog-rest-service/src/main/java/org/openmetadata/catalog/Entity.java

```diff
@@ -203,6 +218,20 @@ public static void deleteEntity(String entity, UUID entityId, boolean recursive)
     dao.delete(entityId, recursive);
   }
 
+  public static <T> EntityRepository<T> getEntityRepositoryForClass(@NonNull Class<T> clazz) {
+    EntityRepository<T> entityRepository = (EntityRepository<T>) CLASS_ENTITY_REPOSITORY_MAP.get(clazz);
+    if (entityRepository == null) {
+      throw EntityNotFoundException.byMessage(
+          CatalogExceptionMessage.entityTypeNotFound(getEntityNameFromClass(clazz)));
+    }
+    return entityRepository;
+  }
+
+  public static <T> EntityRepository<T>.EntityHandler h(@NonNull T entity) {
```

Comment:
h as in handler?

Reply:
Yes, but any letter works. It's like the $ of jQuery. Given that you need to use it everywhere, better to make it short.